pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -206,23 +209,23 @@ Our Regional Partners will be able to see de-identified demographic information
+Code.org promotes Student and Teacher work on our site and social media channels. These profiles and similar testimonials are always published with the permission of the participating Student or Teacher and, if applicable, their parent or legal guardian, or teacher, and may include Personal Information such as the name, likeness and photo or video of the person being profiled.
 
-### Public reporting on student use and performance
+### Public reporting on Student use and performance
 
-We may publish anonymized information about student performance on our tutorials and Websites, however, we will never publicly disclose your Personal Information in these reports. Data about student performance will remain anonymized, but aggregated, anonymized data over large populations of students may be reported by demographic criteria such as age, general location, gender, ethnicity, and socioeconomic status.
+We may publish de-identified information about Student performance on our tutorials and Websites, however, we will never publicly disclose your Personal Information in these reports. Data about Student performance will remain de-identified, but aggregated, de-identified data over large populations of Students may be reported by demographic criteria such as age, general location, gender, ethnicity, and socioeconomic status.
 
-### We may share anonymous or aggregate data to improve our services and learn more about our Users
+### We may share de-identified or aggregate data to improve our services and learn more about our Users
 
-In certain occasions, Code.org may work with third parties (such as universities and education research organizations) to improve our services or offerings. We may disclose automatically collected and other aggregate non-personal information to authorized partners to conduct research on online education or assist in understanding the usage, viewing, and demographic patterns for certain programs, content, services, promotions, and/or functionality on the Website. We require any research partner that receives anonymous or de-identified data from us to agree in advance that they will not attempt to use this data to identify our Users.
+In certain occasions, Code.org may work with third parties (such as universities and education research organizations) to improve our services or offerings. We may disclose automatically collected and other aggregated and de-identified non-personal information to authorized partners to conduct research on online education or assist in understanding the usage, viewing, and demographic patterns for certain programs, content, services, promotions, and/or functionality on the Website. We require any research partner that receives de-identified data from us to agree in advance that they will not attempt to use this data to identify our Users.
```

Comment:
Does non-personal information not need to be capitalized because it's not personal information?

Reply:
We say non-personal information in 3 places so should be consistent across.